src/pip/_vendor/pep517/build.py

```diff
@@ -3,7 +3,7 @@
 import argparse
 import logging
 import os
-from pip._vendor import toml
+from pip._vendor import tomli
```

Comment:
Woops, I now realize this happens in another vendored package.

We have three options I imagine:
- keep vendoring `toml` package for `pep517` package to use
- use `tomli` here and document that as a change in the vendored `pep517`
- ask if `pep517` maintainers are willing to switch to `tomli` as well, and keep this as is

Reply:
Hmm. `pep517` maintainers == you pypa folks, haha. So I assume you may be somewhat ok if I make a PR in that repo too.

Reply:
Ok there's a roadblock. `pep517` still supports EOL python versions all the way down to Python 2.7.

`tomli` is Python >= 3.6 only.

Should we drop support for EOL pythons in `pep517`?

Reply:
Actually, we don't use `pep517.build`, so we can probably ignore this.

Any further discussions about pep517 dropping support for EOL pythons, or switching to depend on tomli, should happen on that project's tracker - it's a different project with different maintainers even though it's also under the PyPA banner. (There is some overlap, though, so you'll probably see the same people popping up, I'm also a maintainer on pep517, for example).